iceoryx_posh/include/iceoryx_posh/popo/wait_set.hpp

```diff
@@ -16,19 +16,17 @@
 
 #include "iceoryx_posh/iceoryx_posh_types.hpp"
 #include "iceoryx_posh/internal/popo/building_blocks/condition_variable_waiter.hpp"
-#include "iceoryx_posh/popo/condition.hpp"
-#include "iceoryx_posh/runtime/posh_runtime.hpp"
 #include "iceoryx_utils/cxx/vector.hpp"
 
```

Comment:
should we make the waitset a template with the max number of conditions the template argument. And maybe MAX_NUMBER_OF_CONDITIONS_PER_WAITSET as default?

Reply:
I will consider this in the upcoming NextGen pull request: https://github.com/ApexAI/iceoryx/tree/iox-%23341-waitset-next-gen